apps/site/components/Downloads/Release/ReleaseCodeBox.tsx

```diff
@@ -28,14 +29,32 @@ const ReleaseCodeBox: FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [release.versionWithPrefix, os, platform]);
 
+  const parseVersion = () => {
+    const version = release.versionWithPrefix.replace('v', '');
+    const versionParts = version.split('.');
+    return {
+      major: parseInt(versionParts[0]),
+      minor: parseInt(versionParts[1]),
+      patch: parseInt(versionParts[2]),
+    };
+  };
+
   const codeLanguage = os === 'WIN' ? 'PowerShell' : 'Bash';
 
   return (
     <div className="mb-2 mt-6 flex flex-col gap-2">
```

Comment:
Maybe we should put these styles in a css module to make them cleaner?

Reply:
@AugustinMauroy Augustin, not sure what is meant by "alert box" can you clarify?

Also, not sure if changing to a CSS module is required or not. I'm not familiar with what that would entail.

Reply:
my question about css module is more for the website team. And alert box mean red box. But you should use Banner component like Claudio said in this [comment](https://github.com/nodejs/nodejs.org/pull/7086#issuecomment-2395559841)